lib/app/service/ntp_service.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:app/app/service/service.dart';
+import 'package:injectable/injectable.dart';
+import 'package:logging/logging.dart';
+import 'package:ntp/ntp.dart';
+import 'package:rxdart/rxdart.dart';
+
+/// Maximum time to wait for NTP server response
+const _updateTimeout = Duration(seconds: 10);
+
+/// Period of time between NTP server requests
+const _updatePeriod = Duration(minutes: 1);
+
+/// Delay between app resume and NTP server request
+const _resumedDelay = Duration(milliseconds: 500);
+
+Timer? _updateTimer;
+
+/// Service that helps watching app lifecycle state.
```

Comment:
I think , wrong description from `AppLifecycleService `

Reply:
Yeah, thank you!